utils.py

```diff
@@ -152,7 +152,32 @@ def is_empty(_dir: Path) -> bool:
 
 
 def replace_chars(val):
-    val = " ".join(val.split())
-    val = val.replace("™", "'")
-    val = val.replace("Œ", "-")
-    return val
+    replacements = [('Œ', '-'),
```

Comment:
Looks good. Thanks for investigating this long list of characters that get translated oddly.